src/sfr.cpp

```diff
@@ -147,13 +147,17 @@ namespace sfr {
         // OP Codes 1900
         SFRField<bool> fire = SFRField<bool>(false, 1900);
         SFRField<bool> arm = SFRField<bool>(false, 1901);
-        SFRField<uint16_t> attempts = SFRField<uint16_t>(0, 1902);
+        SFRField<uint16_t> regular_attempts = SFRField<uint16_t>(0, 1902);
         SFRField<uint16_t> camera_attempts = SFRField<uint16_t>(0, 1904);
-        SFRField<uint32_t> start_time = SFRField<uint32_t>(0, 1903);
+        SFRField<uint32_t> mandatoryburn_start_time = SFRField<uint32_t>(0, 1903);
         SFRField<uint32_t> burn_time = SFRField<uint32_t>(500, 1905);
```

Comment:
Looks like this is set for 500 milliseconds which would mean we're only burning for 0.5 seconds then pausing for 0.25 seconds before burning again? Is this right? For our balloon launch, we burned for multiple seconds to ensure the wire got hot enough

Reply:
Talked to Josh, and the 0.5 second burn is correct, but he is concerned that the 0.25 second wait might be too short